org.scilab.Scilab.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: org.scilab.Scilab
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: scilab
+rename-desktop-file: scilab.desktop
+rename-icon: scilab
+rename-appdata-file: scilab.appdata.xml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /include
+  - "*.la"
+  - "*.a"
+
+modules:
+  - name: ncurses
+    no-autogen: true
+    config-opts:
+      - --disable-static
+      - --without-tests
+      - --with-shared
+      - --with-termlib=tinfo
+      - --with-abi-version=5
+    make-install-args:
+      - install.libs
+    cleanup:
+      - /bin
+      - /include
+      - /share/man
+      - "/lib/*.a"
+      - "/lib/*.la"
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/ncurses/ncurses-6.2.tar.gz
+        sha256: 30306e0c76e0f9f1f0de987cf1c82a5c21e1ce6568b9227f7da5b71cbea86c9d
+
+    # Use Scilab binary
+    # Delete smaller icons so rename them is easier.
+  - name: Scilab
+    buildsystem: simple
+    ensure-writable:
+      - /lib
+      - /share
+    build-commands:
+      - chmod +x bin/{scilab,xcos,scilab-bin,scinotes}
+      - cp -rn {bin,lib,share,thirdparty} ${FLATPAK_DEST}
+      - cp -rn COPYING ACKNOWLEDGEMENTS CHANGES.md README.md ${FLATPAK_DEST}/share/scilab
+      - rm -r /app/share/icons/hicolor/{16x16,24x24,32x32,48x48}/
+      - mv /app/share/icons/hicolor/256x256/apps/puffin.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.puffin.png
```

Comment:
I don't see a desktop entry using this.

Reply:
I don't see puffin desktop entry as well, but they include it, som I'm not sure if tey use it for something or not. Should I remove them?